modules/core/02-client/keeper/migrations.go

```diff
@@ -3,8 +3,10 @@ package keeper
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
-	"github.com/cosmos/ibc-go/v8/modules/core/02-client/migrations/v7"
+	v7 "github.com/cosmos/ibc-go/v8/modules/core/02-client/migrations/v7"
```

Comment:
undo

Reply:
lol, I kept having this same problem as well with "redundant import alias" and no matter what I tried my editor kept putting it back on save

Reply:
yea same :smile: